src/view/pages/Exports/components/CardSummary/style.ts

```diff
@@ -0,0 +1,11 @@
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+  info: {
+    textAlign: 'center',
+    padding: 5,
+  },
+  label: {
+    fontWeight: 'bold',
```

Comment:
I'm not sure if it can be, but if possible use `H1`, etc to enforce font weight, size etc

Reply:
The trouble with using h1 etc is that they will modify attributes other than just the fontWeight. For consistency I matched this styling to Bea's CardSummary components so if we want to change this we should reflect that across all of these components.

Reply:
Alright sounds good :+1: